niworkflows/utils/spaces.py

```diff
@@ -476,6 +477,14 @@ def get_fs_spaces(self):
                 if s.name in ("fsaverage", "fsnative")]
 
 
+class StoreSpacesAction(argparse.Action):
+    def __call__(self, parser, namespace, values, option_string=None):
+        spaces = getattr(namespace, self.dest) or []
+        for val in values:
+            spaces.extend(Space.from_string(val))
+        setattr(namespace, self.dest, spaces)
+
+
```

Comment:
why is this necessary? it seems argparse correctly handles ``Space.from_string`` alone?

Reply:
i think it correctly handled singleton cases, but would fail when multiple specs where given, i.e. `fsaverage:den-10k:den-164k`